src/simcore_service_deployment_agent/cmd_utils.py

```diff
@@ -38,7 +38,13 @@ async def run_cmd_line(cmd: List[str], cwd_: str = ".") -> str:
 
 async def run_cmd_line_unsafe(cmd: str, cwd_: str = ".") -> str:
     proc = await asyncio.create_subprocess_shell(
-        cmd, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE, cwd=cwd_
+        cmd,
+        stdout=asyncio.subprocess.PIPE,
+        stderr=asyncio.subprocess.PIPE,
+        cwd=cwd_,
+        env={
+            "PWD": cwd_,
+        },
     )
```

Comment:
Question1: what is the idea behind having these 2 functions (run_cmd_line_unsafe vs run_cmd_line)? When do you use one or the other?
Question2: not sure I understand the fix here. so you override PWD by "." ? is this not supposed to be an absolute path? you mean that the ```cwd``` argument does not work? (if yes then what about the call above?) any additional information on that?

Reply:
1: discussed in person
2: you are right, it is removed